<launch>
    <rosparam file="$(find heart_beat_proj)/config/endoscope.yaml" command="load"/>

    <node pkg="heart_beat_proj" name="video_to_ros" type="video_to_ros.py" output="screen">
        <param name="original_video"
            value="/home/corbe/heart_beat_ws/src/heart_beat_proj/videos/video2.1.mp4" />
    </node>
</launch>
